api/v1alpha1/jvmchaos_types.go

```diff
@@ -0,0 +1,294 @@
+type JVMOOMSpec struct {
+	// JVMCommonParmeter represents the common jvm chaos parmeter
+	JVMCommonParmeter `json:",inline"`
+
+	// Area represents JVM memory area, currently supported [HEAP, NOHEAP, OFFHEAP], required.
+	// Eden+Old is denoted by HEAP
+	// Metaspace is denoted by NOHEAP
+	// off-heap memory is denoted by OFFHEAP
+	// +kubebuilder:validation:Enum=HEAP;NOHEAP;OFFHEAP
+	Area string `json:"area"`
+
+	// Block represents specifies the size of the object that supports only the HEAP and OFFHEAP areas in MB
+	// +optional
+	Block string `json:"block"`
+
+	// Interval represents unit MS, default interval between 500 OOM exceptions, only in non-violent mode, can slow down the frequency of GC without worrying about the process being unresponsive
+	// +optional
+	Interval int `json:"interval"`
+
+	// WildMode represents default false, whether to turn on wild mode or not.
+	// If it is wild mode, the memory created before will not be released after OOM occurrence, which may cause the application process to be unresponsive
+	// +optional
+	WildMode bool `json:"wildmode"`
+}
+
+// JVMCpufullloadSpec represents the detail about JVM chaos action of CPU is full
+type JVMCpufullloadSpec struct {
+	// JVMCommonParmeter represents the common jvm chaos parmeter
+	JVMCommonParmeter `json:",inline"`
+
+	// CpuCount represents the number of CPU cores to bind to, that is, specify how many cores are full
+	CpuCount int `json:"cpucount"`
+}
+
+// JVMScriptSpec represents the detail about JVM chaos action of Java or Groovy scripts
+type JVMScriptSpec struct {
+	// JVMCommonParmeter represents the common jvm chaos parmeter
+	JVMCommonParmeter `json:",inline"`
+
+	// Content represents the script content is Base64 encoded content.
+	// Note that it cannot be used with file
+	// +optional
+	Content string `json:"content"`
+
+	// File represents script file, absolute path to file
+	// Note that it cannot be used with content
+	// +optional
+	File string `json:"file"`
+
+	// Name represents script name, use for logging
+	// +optional
+	Name string `json:"name"`
+
+	// Type represents script type, java or groovy, default to java
+	// +kubebuilder:validation:Enum=java;groovy;
+	// +optional
+	Type string `json:"type"`
+}
+
+// JVMReturnSpec
+type JVMReturnSpec struct {
+	// JVMCommonParmeter represents the common jvm chaos parmeter
+	JVMCommonParmeter `json:",inline"`
+
+	// Value represents specifies the return value of a class method, supporting only primitive, null, and String return values. required
+	Value string `json:"value"`
+}
+
+type JVMDelaySpec struct {
+	// JVMCommonParmeter represents the common jvm chaos parmeter
+	JVMCommonParmeter `json:",inline"`
+
+	// Time represents delay time, in milliseconds, required
+	Time int `json:"time"`
+
+	// Offset represents delay fluctuation time
+	// +optional
+	Offset int `json:"offset"`
+}
+
+// JVMCommonParmeter represents the common jvm chaos parmeter
```

Comment:
```suggestion
// JVMCommonParmeter represents the common jvm chaos parameter
```

Reply:
and have many other typos for `parmeter`

Reply:
All done! 
PTAL! Thanks!